src/Data/String/CodePoints.purs

```diff
@@ -251,12 +375,24 @@ takeFallback n s = case uncons s of
 -- | Returns a string containing the leading sequence of code points which all
 -- | match the given predicate from the string. Operates in constant space and
 -- | in time linear to the length of the string.
+-- |
+-- | ```purescript
+-- | >>> takeWhile (\c -> codePointToInt c == 0x1D400) "𝐀𝐀 b c 𝐀"
+-- | "𝐀𝐀" 
+-- | ```
+-- |
 takeWhile :: (CodePoint -> Boolean) -> String -> String
 takeWhile p s = take (count p s) s
 
 
 -- | Creates an array of code points from a string. Operates in space and time
 -- | linear to the length of the string.
+-- |
+-- | ```purescript
+-- | >>> map singleton (toCodePointArray "b 𝐀𝐀 c 𝐀")
```

Comment:
Do you think this would be clearer if we showed the actual array first?

```purescript
>>> codePointArray = toCodePointArray "b 𝐀𝐀 c 𝐀"
>>> codePointArray
[CodePoint 0x62, ...]
>>> map singleton codePointArray
["b", " ", ...]
```